src/Components/Aspire.Keycloak/Aspire.Keycloak.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <PackageTags>$(ComponentCommonPackageTags) Keycloak</PackageTags>
+    <PackageIconFullPath>$(SharedDir)Keycloak_256x.png</PackageIconFullPath>
```

Comment:
We'll need to reach out to Keycloak to get specific permission to use the icon before including it here. Please remove this for now and log a separate issue to follow up once this is merged.

Reply:
Removed!